burn-compute/src/tune/operation.rs

```diff
@@ -3,6 +3,18 @@ use alloc::string::String;
 use alloc::vec::Vec;
 use core::fmt::{Debug, Display};
 use core::hash::Hash;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
+/// Default checksum for an operation set
+#[cfg(feature = "autotune-persistent-cache")]
+pub fn compute_checksum(autotunables: &[Box<dyn AutotuneOperation>]) -> String {
+    let mut checksum = String::new();
+    autotunables.iter().for_each(|op| {
+        checksum += op.name();
+    });
+    format!("{:x}", md5::compute(checksum))
```

Comment:
I wonder if the checksum type can be an integer (u64)?

Reply:
We can do it if we are OK with loosing half of the hash value.

Reply:
We decided to keep it as a string.